docs/example-scenario/apps/devops-with-aks.md

```diff
@@ -47,7 +47,7 @@ This scenario covers a DevOps pipeline for a Node.js web application and databas
 
 ### Alternatives
 
-* [Visual Studio Team Services][vsts] and Team Foundation Server help you implement a continuous integration (CI), test, and deployment (CD) pipeline for any app.
+* [Azure Pipelines][azure-pipelines] help you implement a continuous integration (CI), test, and deployment (CD) pipeline for any app.
```

Comment:
Seems awkward to separate the CI from the CD in this sentence. Maybe this instead?
"Azure Pipelines let you create a continuous integration and continuous delivery (CI/CD) pipeline for any app."